### DISCUSSION TITLE
"Highlight" via Inspect pane

### DISCUSSION THREAD

**Discussion by zarnejo-exers** [gama-platform/gama.old#3932] [Q&A]
Greetings everyone!

I would like to ask about the "highlight" feature in GAMA 1.9.2. 
In GAMA 1.8.2, I can highlight the agent either through the visualization pane or through the inspect pane. However, in GAMA 1.9.2, though I can still highlight agents through the visualization pane, I cannot find the feature any more in the inspect pane. 

<img width="463" alt="Screenshot 2023-11-14 at 15 30 09" src="https://github.com/gama-platform/gama/assets/3262396/1bada882-8d4a-41c5-9b5e-e8985c192dca">
<img width="481" alt="Screenshot 2023-11-14 at 15 39 04" src="https://github.com/gama-platform/gama/assets/3262396/e36059c9-3188-4dcb-918f-cc26807cd712">

I'm not sure if this is a design decision, but I find the highlight feature via inspect useful especially when an agent has neighbors and I want to check if these neighbors are the correct ones. Focus can be useful, but in my case, I find the "highlight" feature providing the exact information I need to verify my code. In this regard, is it possible to have the feature accessible via the inspect pane again? Thank you in advance for your response!

PS.
I used: GAMA_1.9.2_MacOS_M1_with_JDK.

**Comment by lesquoyb** [comment]
The issue has been solved and will be available in the next release 1.9.3
